distribution/changelog.txt

```diff
@@ -19,6 +19,7 @@
 - Improved: [#2989] Multiplayer window now changes title when tab changes.
 - Improved: [#5339] Change eyedropper icon to actual eyedropper and change cursor to crosshair.
 - Improved: Raising land near the map edge makes the affected area smaller instead of showing an 'off edge map' error.
+- Fix: [#7341] Change staff placement to only consider guests who are on path tiles
```

Comment:
Put this line between fix 7327 and fix 7382 to keep this list sorted. It's common practice to use the title of the addressed issue here.